vumi_twilio_api/twiml_parser.py

```diff
@@ -1,15 +1,47 @@
+import re
 import xml.etree.ElementTree as ET
 
 
 class Verb(object):
     """Represents a single verb in TwilML. """
+    name = "Verb"
 
-    def __init__(self, verb, attributes={}, nouns={}):
-        self.verb = verb
+    def __init__(self, attributes={}, nouns=[]):
         self.attributes = attributes
         self.nouns = nouns
 
 
+class Play(Verb):
+    """Represents the Play verb"""
+    name = "Play"
+    digits_re = re.compile('[0-9w]')
+
+    @classmethod
+    def from_xml(cls, xml):
+        """Returns a new Play Verb given an ElementTree object"""
+        nouns = [xml.text]
```

Comment:
Given the structure of XML, I wonder if this concept of ``nouns`` is the best idea. For Play this is more like an extra attribute, or perhaps just content. If you agree I suggest we leave refactoring this for a possible later PR though.